src/listing.rs

```diff
@@ -78,6 +80,24 @@ pub enum EntryType {
     File,
 }
 
+#[derive(Clone, Copy, PartialEq, Eq)]
+/// Possible entry size types
+pub enum EntrySize {
+    /// EntryCount is number of entries in a directory
+    EntryCount(usize),
+    /// Bytes is number of bytes in a file
+    Bytes(bytesize::ByteSize),
+}
+
+impl Display for EntrySize {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            EntrySize::EntryCount(count) => write!(f, "{}", count),
+            EntrySize::Bytes(bytes) => write!(f, "{}",  bytes),
+        }
+    }
+}
+
```

Comment:
Wouldn't it be cool to also show the combined directory file size?

Reply:
Not sure what you are referring to by "combined directory file size". Are you referring to the total size in bytes of a directory? That might be a bit expensive and probably would require a persistent cache to be anywhere near fast.

Reply:
Yes, the byte sizes. I recognize it might be too slow. Hm. If you like to, you could try to experiment with this.